src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -210,16 +219,24 @@ protected int doWork() {
             ++total;
             final Interval source = new Interval(ctx.getContig(), ctx.getStart(), ctx.getEnd(), false, ctx.getContig() + ":" + ctx.getStart() + "-" + ctx.getEnd());
             final Interval target = liftOver.liftOver(source, LIFTOVER_MIN_MATCH);
+
+            if (target == null) {
+                rejectVariant(ctx, FILTER_NO_TARGET);
+                continue;
+            }
+            if (ctx.getReference().length() != target.length()){
```

Comment:
Is it always the case that this means that the indel spans multiple chain intervals?  I'm not sure I see why that is.  Could you please add a comment to the code explaining what you are testing for here?